bedrock/mozorg/templates/mozorg/mission.html

```diff
@@ -73,7 +73,7 @@ <h1 class="title-shadow-box">{{ _('We’re building a better Internet') }}</h1>
       <p>{{ _('Topics include support, products, and technologies') }}</p>
     </li>
     <li>
-      <h4><a href="/about/governance.html">{{ _('Governance') }}</a></h4>
+      <h4><a href="/about/governance/">{{ _('Governance') }}</a></h4>
```

Comment:
I was wondering about that. I did not actually mean to change this one as it works fine on production. I assume .htaccess rules kicking in. But hey, let me just fix them to use those names.